tests/augmentations/test_color_temperature.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import numpy as np
+import random
+from discolight.annotations import (annotations_to_numpy_array)
+from discolight.augmentations.colortemperature import ColorTemperature
+
+@pytest.mark.usefixtures("sample_image")
+def test_my_augmentation_warmer(sample_image):
+    img, annotations = sample_image    
+
+    bboxes = annotations_to_numpy_array(annotations)
+    
+    # Kelvin values smaller than 6600K map to RGB multipliers 
+    # acting as warming filters.
+    kelvin_input = random.randint(1000, 6560)
+    
+    augmentation = ColorTemperature(kelvin=kelvin_input)
+    
+    aug_img, aug_bboxes = augmentation.augment(img.copy(), bboxes.copy())
+    
+    # Warming filters make reds more intensive and blue less.
+    assert np.less_equal(
+        img[:,:,0], aug_img[:,:,0]
+    ).all(), "Performing augmentation does not yield expected red channel"
+    assert np.greater_equal(
+        img[:,:,2], aug_img[:,:,2]
+    ).all(), "Performing augmentation does not yield expected blue channel"
+    assert np.array_equal(
+        bboxes, aug_bboxes
+    ), "Performing augmentation does not yield original augmentation"
```

Comment:
The assert message should state that you are checking the bounding boxes
```suggestion
    ), "Performing augmentation does not yield original bounding boxes"
```